File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/GamepadUtil.java
package org.firstinspires.ftc.teamcode;

import com.qualcomm.robotcore.hardware.Gamepad;

public class GamepadUtil {
    public static boolean leftTriggerPressed(Gamepad gamepad) {
        return gamepad.left_trigger < 0.1;
    }
    public static boolean rightTriggerPressed(Gamepad gamepad) {
        return gamepad.right_trigger < 0.1;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/GamepadUtil.kt
package org.firstinspires.ftc.teamcode

import com.qualcomm.robotcore.hardware.Gamepad

TODO("TEST")

val Gamepad.leftTriggerPressed get() = left_trigger > 0.85

val Gamepad.rightTriggerPressed get() = right_trigger > 0.85
